src/js/actions/layers.js

```diff
@@ -64,44 +66,41 @@ define(function (require, exports) {
         "layerLocking",
         "itemIndex",
         "background",
-        "bounds",
-        "boundsNoMask",
-        "boundsNoEffects",
-        "mode"
+        "layerKind",
+        "layerSectionExpanded",
+        "artboardEnabled",
+        "smartObject",
+        "vectorMaskEnabled",
+        "vectorMaskEmpty",
+        "textWarningLevel"
     ];
-
+    
     /**
-     * Basic optional properties to request of layer descriptors from Photoshop.
-     * When initializing documents, these are the only properties requested for
-     * non-selected layers.
+     * Required properties for children of the selected layers
      * 
      * @private
      * @type {Array.<string>} 
      */
-    var _optionalLayerProperties = [
-        "boundingBox",
-        "layerKind",
+    var _lazyLayerBoundProperties = [
+        "layerID", // redundant but useful for matching results
         "artboard",
-        "artboardEnabled",
-        "smartObject",
-        "layerSectionExpanded",
-        "vectorMaskEnabled",
-        "vectorMaskEmpty",
-        "textWarningLevel"
+        "bounds",
+        "boundingBox",
+        "boundsNoMask",
+        "boundsNoEffects",
+        "pathBounds"
```

Comment:
The `_lazyLayerBoundProperties` is no longer used and the bounds name is determined by `Bounds.getRequiredBoundsName`.